webdriver/bidi.json

```diff
@@ -1083,6 +1083,74 @@
           }
         }
       },
+      "emulation": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "139"
+            },
```

Comment:
@whimboo Is this feature riding the train, or is it Nightly only for now?

@Elchi3 Assuming this feature rides the train and lands in Firefox 139 stable, is it okay to record the version number, or should we systematically put `"preview"` as long as the version _is_ nightly? If so, I would write a auto-fixable lint for this.

Reply:
This new module and command rides the train and will be released in Firefox 139. There is no plan to make it Nightly only.

Reply:
> @Elchi3 Assuming this feature rides the train and lands in Firefox 139 stable, is it okay to record the version number, or should we systematically put `"preview"` as long as the version _is_ nightly? If so, I would write a auto-fixable lint for this.

No, if we anticipate a feature to ride the trains to release, then 139 is the version number to record. 
If a feature is only enabled in Nightly, TP or Canary and doesn't ride the trains, then its status is "preview".

Reply:
Thank you both!

Reply:
PS: If the case comes up that a BiDi feature is implemented in Nightly only, and this should be recorded in BCD, I will probably need to update the Google Apps script, as it only extracts version numbers.